tests/manual/formsubmit.md

```diff
@@ -0,0 +1,7 @@
+## Updating editor element when submitting a form
+
+1. Start simple HTTP server located in `ckeditor5-core/tests/_utils/` by calling `node echo-server.js`.
```

Comment:
TBH, this whole server thing feels like an overkill (although lovely :D). We don't have to assert if the web browser correctly handles POST. I assume, browse vendors check it anyway ;-)

The only thing that we need is the actual textarea (`editor#element`) visible to the user to check if it really updates upon submit. And of course some `preventDefault` to avoid reloading. 

IMO it's all about removing `display: none` from `editor#element`, giving it some styles (`width: 100%`, some height) and explaining what should happen where.